GameData/Skopos/Contracts/Level1Comsats/l1_us_tracking_ships.cfg

```diff
@@ -12,8 +12,8 @@ CONTRACT_TYPE {
   sortKey = 280
   title = US Tracking Ships - Level 1
   // Mostly pilfered from NASA SP-32 p. 740.
-  description = <b>Program: First Generation Telecommunications Satellites (Skopos Beta)<br>Type: <color=blue>Optional</color></b><br><br>We would like to create "Ground stations at sea", ships equipped with unified S-band antennas to relay data from manned and unmanned scientific spacecraft. However, a ship at sea cannot have a landline connection to mission control, and existing HF radio systems are unreliable and low quality. Instead, these ships have had a satellite communications terminal installed, allowing them to relay data back to mission control via communications satellites.
```

Comment:
"Ground stations at sea" is actually correct here, both because it is quoting something (which I should have referenced) and because the primary purpose of these ships was to function as tracking stations.

Reply:
Fixed